client/gutenberg/extensions/publicize/connection-verify.jsx

```diff
@@ -63,8 +66,8 @@ class PublicizeConnectionVerify extends Component {
 	 */
 	connectionTestStart = () => {
 		apiFetch( { path: '/wpcom/v2/publicize/connection-test-results' } )
-			.then( () => this.connectionTestComplete )
-			.catch( () => this.connectionTestRequestFailed );
```

Comment:
These were the main broken bits, is that right? The methods were not being invoked?

Reply:
That, and also the wrong fields were used when rendering 😅 For example `refreshText` instead of the proper `refresh_text`.